.github/CHANGELOG.md

```diff
@@ -2,26 +2,26 @@
 
 <h3>New features since last release</h3>
 
-* Adds a device test suite, located at `pennylane/plugins/tests`, which can be used 
-  to run generic tests on core or external devices calling 
-  
-  >>> pytest pennylane/plugins/tests --device default.qubit --shots 1234 --analytic False                                                                                                                                                                                                                                                                    
-  
+* Adds a device test suite, located at `pennylane/plugins/tests`, which can be used
+  to run generic tests on core or external devices calling
+
+  >>> pytest pennylane/plugins/tests --device default.qubit --shots 1234 --analytic False
```

Comment:
Might be good to surround with `pycon` as it's currently rendering as:
![image](https://user-images.githubusercontent.com/49409390/86786197-b7ee4e00-c031-11ea-922b-c4bd7f682ee2.png)

Sorry I couldn't make a direct suggestion as the triple backticks don't work with github suggestions.

Reply:
Oh! This came from a previous PR, but happy to change it. Not sure why it's showing up in this PR :/